Sources/App/Core/Search.swift

```diff
@@ -25,6 +33,14 @@ enum Search {
             case repositoryOwner = "owner"
             case summary
         }
+
+        var asRecord: Record {
+            .init(packageId: packageId,
+                  packageName: packageName,
+                  repositoryName: repositoryName,
+                  repositoryOwner: repositoryOwner,
+                  summary: summary)
+        }
```

Comment:
So this probably looks a bit convoluted - the issue is that we need to set the `packageName` key to _decode_ as `package_name` because we're also using `Record` to decode the results from the db.

In order to encode it for the API we're then stuck with the snake case unless we map to a distinct type. Which is arguably the cleaner thing to do, although it leads to these pass-through gymnastics.

Another thing we _could_ do is change the search query to set a field alias `package_name AS packageName` with a migration.

This here is probably better, isn't it?